### DISCUSSION TITLE
Being able to filter traffic from Plausible UI

### DISCUSSION THREAD

**Discussion by gperriard** [plausible/analytics#2194] [Anything Else]
Hello!

Is it currently possible to include only hits coming from trusted domains on Plausible?

So far, I could find only two ways to restraint domains from sending hits to Plausible:
* Removing Plausible on those domains
* Adding `localStorage.plausible_ignore=true` before the Plausible script on those domains to avoid HTTP request being made to Plausible servers
* Asking people visiting those domains to configure uBlocker to blocking requests going to Plausible servers

We are loading Plausible on multiple environments (testing and production).
On testing, we test every new event tracking setup by looking at the HTTP requests.
Using technics above aren't sustainable in my case, as I can enforce everyone on the project to adapt their browsers. I also can't add `localStorage.plausible_ignore=true` before the loading of Plausible because I need to test events (checking HTTP request) on testing server.

I can also imagine that the integration of data in Plausible can be compromised as it can be flooded with data from other domains by a malicious person or by inadvertence if someone put the snippet on a wrong domain.

Other dimensions could also be used in the filtering to handle the following case:
* Exclude/include user agents
* Exclude/include pages
* Exclude/Include countries
...

Looking forward to all the feedback! :)

**Comment by metmarkosaric** [comment]
thanks for the feedback @gperriard! and yes, you're right. the options you listed are currently the only options we have. would be nice to allow more flexibility for this in the future

**Comment by metmarkosaric** [comment]
**Update**: It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding

**Reply by gperriard** [reply]
Hello @metmarkosaric
Thank you for adding this feature within Plausible ! :)
